Dapper/SqlMapper.cs

```diff
@@ -91,12 +91,12 @@ private static void CollectCacheGarbage()
         private static int collect;
         private static bool TryGetQueryCache(Identity key, out CacheInfo value)

```

Comment:
`[NotNullWhen(true)]`

Reply:
I think the issue there was TFMs, but I guess I can declare them as file-local or something when needed

Reply:
I did this in SE.Redis FWIW, think I just declared attribute based on TFM